docs/install.md

```diff
@@ -66,3 +66,21 @@ You are now ready to configure Tekton Triggers for your workflow. For instructio
 
 - [Tekton Triggers Getting Started Guide](./getting-started/)
 - [Tekton Triggers code examples](https://github.com/tektoncd/triggers/tree/main/examples)
+
+## Customizing the Triggers Controller behavior
+
+To customize the behavior of the Triggers Controller, modify the ConfigMap `feature-flags` as follows:
+
+- `enable-api-fields`: set this flag to "stable" to allow only the
+most stable features to be used. Set it to "alpha" to allow alpha
+features to be used.
```

Comment:
This probably needs some rewording / copy+paste fixing - here this mentions stable vs alpha features, which makes me think these are triggers features, but below this says "allow alpha features to be used in Tasks/Pipelines" which makes me think it's the API version being used for the Pipelines API.

If it's the former, let's fix the comment below and add more detail on what features this is enabling.

If it's the latter, should we rename this flag to `pipelines-api-version` or something similar?

In either case, we should also detail the default behavior if the flag is not explicitly set.

Reply:
The docs were copied from the pipelines' docs/install.md so the typo. Fixed and added a bit about it defaulting to stable.